File: scripts/check-db.py
import pymongo
import sys

MONGODB_URI = "mongodb+srv://pminsights:pminsights@cluster0.ruitbsk.mongodb.net/BhookBuster?retryWrites=true&w=majority&appName=Cluster0"
DB_NAME = "BhookBuster"

try:
    client = pymongo.MongoClient(MONGODB_URI)
    db = client[DB_NAME]
    
    print("Connected to MongoDB Atlas!")
    
    restaurants_col = db["restaurants"]
    riders_col = db["riders"]
    
    restaurants = list(restaurants_col.find({}))
    print(f"\n--- Restaurants found: {len(restaurants)} ---")
    for idx, r in enumerate(restaurants):
        is_verified = r.get("isVerified")
        print(f"{idx + 1}. Name: {r.get('name')} | isVerified: {is_verified} ({type(is_verified)}) | Owner: {r.get('ownerId')}")
        
    riders = list(riders_col.find({}))
    print(f"\n--- Riders found: {len(riders)} ---")
    for idx, r in enumerate(riders):
        is_verified = r.get("isVerified")
        print(f"{idx + 1}. UserID: {r.get('userId')} | isVerified: {is_verified} ({type(is_verified)}) | Phone: {r.get('phoneNumber')}")

except Exception as e:
    print(f"Error connecting or querying: {e}", file=sys.stderr)

File: scripts/check-db.js
import { MongoClient } from "mongodb";

const MONGODB_URI = "mongodb+srv://pminsights:pminsights@cluster0.ruitbsk.mongodb.net/BhookBuster?retryWrites=true&w=majority&appName=Cluster0";
const DB_NAME = "BhookBuster";

async function run() {
  const client = new MongoClient(MONGODB_URI);
  try {
    await client.connect();
    console.log("Connected to MongoDB!");
    const db = client.db(DB_NAME);

    const restaurantsCollection = db.collection("restaurants");
    const ridersCollection = db.collection("riders");

    const restaurants = await restaurantsCollection.find({}).toArray();
    console.log(`\n--- Restaurants found: ${restaurants.length} ---`);
    restaurants.forEach((r, i) => {
      console.log(`${i + 1}. Name: ${r.name} | isVerified: ${r.isVerified} (${typeof r.isVerified}) | Owner: ${r.ownerId}`);
    });

    const riders = await ridersCollection.find({}).toArray();
    console.log(`\n--- Riders found: ${riders.length} ---`);
    riders.forEach((r, i) => {
      console.log(`${i + 1}. Picture: ${r.picture ? "Exists" : "None"} | isVerified: ${r.isVerified} (${typeof r.isVerified}) | User: ${r.userId}`);
    });

  } catch (err) {
    console.error("Error running script:", err);
  } finally {
    await client.close();
  }
}

run();
